src/tasks/shop-admin/ExpectNotification.ts

```diff
@@ -0,0 +1,20 @@
+import { test as base } from '@playwright/test';
+import type { Task } from '../../types/Task';
+import type { FixtureTypes} from '../../types/FixtureTypes';
+
+export const ExpectNotification = base.extend<{ ExpectNotification: Task }, FixtureTypes>({
+    ExpectNotification: async ({ ShopAdmin }, use ) => {
+        const task = (message: string, options: { close: boolean } = { close: true }) => {
```

Comment:
As there is only one option, I would prefer exposing the close option as a direct function parameter.